go.mod

```diff
@@ -4,6 +4,8 @@ go 1.21
 
 replace github.com/jpillora/overseer => github.com/trufflesecurity/overseer v1.2.7
 
+replace github.com/snowflakedb/gosnowflake => github.com/trufflesecurity/gosnowflake v0.0.1
```

Comment:
I see [trufflesecurity/gosnowflake](https://github.com/trufflesecurity/gosnowflake) is forked from `snowflakedb/gosnowflake` with a lot of code deleted. What is the affect of doing that?

Also, how much effort do you anticipate it'll be for us to maintain this fork?